docs/maintain-polkadot-parameters.md

```diff
@@ -11,9 +11,9 @@ at the [chain state](https://polkadot.js.org/apps/#/chainstate/constants) and/or
 
 ### Periods of common actions and attributes
 
-_NOTE: Kusama generally runs 4x as fast as Polkadot, except in the time slot duration itself. See
-[Polkadot Parameters](maintain-polkadot-parameters.md) for more details on how Kusama's parameters
-differ from Polkadot's._
+_NOTE: Polkadot generally runs at &frac14;th the speed of Kusama, except in the time slot duration itself. See
+[Kusama Parameters](https://guide.kusama.network/docs/kusama-parameters) for more details on how Polkadot's parameters
```

Comment:
For the Kusama Parameters page, we can just change it to "kusama-parameters.md" and it will reference the page the same way it referenced "maintain-polkadot-parameters.md"

Reply:
ah! thanks.